nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/test/java/org/apache/nifi/processors/azure/storage/queue/TestPutAzureQueueStorage_v12.java

```diff
@@ -0,0 +1,175 @@
+        runner.setProperty(PutAzureQueueStorage_v12.VISIBILITY_TIMEOUT, "30 secs");
+
+        ProcessContext processContext = runner.getProcessContext();
+        Collection<ValidationResult> results = new HashSet<>();
+        if (processContext instanceof MockProcessContext) {
+            results = ((MockProcessContext) processContext).validate();
+        }
+
+        assertEquals(1, results.size());
+        Iterator<ValidationResult> iterator = results.iterator();
+        assertTrue(iterator.next().getExplanation().contains("positive number"));
+    }
+
+    @Test
+    public void testValidRequestTimeout() {
+        runner.setProperty(PutAzureQueueStorage_v12.QUEUE, "dummyqueue");
+        runner.setProperty(PutAzureQueueStorage_v12.TTL, "7 days");
+        runner.setProperty(PutAzureQueueStorage_v12.VISIBILITY_TIMEOUT, "30 secs");
+        runner.setProperty(PutAzureQueueStorage_v12.REQUEST_TIMEOUT, "15 secs");
+
+        ProcessContext processContext = runner.getProcessContext();
+        Collection<ValidationResult> results = new HashSet<>();
+        if (processContext instanceof MockProcessContext) {
+            results = ((MockProcessContext) processContext).validate();
+        }
+
+        assertEquals(0, results.size());
+    }
+
+    @Test
+    public void testInvalidRequestTimeoutZeroSecs() {
+        runner.setProperty(PutAzureQueueStorage_v12.QUEUE, "dummyqueue");
+        runner.setProperty(PutAzureQueueStorage_v12.TTL, "7 days");
+        runner.setProperty(PutAzureQueueStorage_v12.VISIBILITY_TIMEOUT, "30 secs");
+        runner.setProperty(PutAzureQueueStorage_v12.REQUEST_TIMEOUT, "0 secs");
+
+        ProcessContext processContext = runner.getProcessContext();
+        Collection<ValidationResult> results = new HashSet<>();
+        if (processContext instanceof MockProcessContext) {
+            results = ((MockProcessContext) processContext).validate();
+        }
+
+        assertEquals(1, results.size());
+        Iterator<ValidationResult> iterator = results.iterator();
+        assertTrue(iterator.next().getExplanation().contains("0 secs"));
+    }
+
+    @Test
+    public void testInvalidRequestTimeoutMoreThanThirtySecs() {
+        runner.setProperty(PutAzureQueueStorage_v12.QUEUE, "dummyqueue");
+        runner.setProperty(PutAzureQueueStorage_v12.TTL, "7 days");
+        runner.setProperty(PutAzureQueueStorage_v12.VISIBILITY_TIMEOUT, "30 secs");
+        runner.setProperty(PutAzureQueueStorage_v12.REQUEST_TIMEOUT, "31 secs");
+
+        ProcessContext processContext = runner.getProcessContext();
+        Collection<ValidationResult> results = new HashSet<>();
+        if (processContext instanceof MockProcessContext) {
+            results = ((MockProcessContext) processContext).validate();
+        }
+
+        assertEquals(1, results.size());
+        Iterator<ValidationResult> iterator = results.iterator();
+        assertTrue(iterator.next().getExplanation().contains("30 secs"));
```

Comment:
This message check is probably not needed, just checking validity is sufficient.